packages/webapp/src/pages/Environment/Settings.tsx

```diff
@@ -115,7 +115,17 @@ export const EnvironmentSettings: React.FC = () => {
             });
             setWebhookUrl(webhook_settings.primary_url);
             setWebhookUrlSecondary(webhook_settings.secondary_url);
+        } else {
+            setWebhookCheckboxSettings({
+                alwaysSendWebhook: false,
+                sendAuthWebhook: false,
+                sendRefreshFailedWebhook: false,
+                sendSyncFailedWebhook: false
+            });
+            setWebhookUrl('');
+            setWebhookUrlSecondary('');
         }
+
```

Comment:
would it make sense to have those settings always returned by the api?

Reply:
@TBonnin I'm pretty sure we could always return them. We would have to either:

1. pre-set an `ExternalWebhook` in the DB when we create the environment 
2. remove the `id` field from the `webhook_settings` part of the response so we can send an empty state without an id 

Technically we could make the id nullable but it seems like it'd be better just to not leak that id since we don't use it to update the webhook anyway.

Reply:
@TBonnin I'll let this sit until tomorrow and we can see what @bodinsamuel says about it. I 100% agree with you about choosing option 2 though.

Reply:
I'm going to merge this as-is, and I logged a ticket to come back to for resolving this discussion, but I think this PR solves an existing problem the fastest way we can right now.